### PR TITLE
Added smiley face to the medibot.

### PR DESCRIPTION
**Pull request by Radezolid**
## About the PR
:)
Requested by [WYCI Suggestion](https://discord.com/channels/968983104247185448/1291013833229467749/1291013833229467749).

## Why / Balance
It looks friendlier and people seem to like the idea of a smiley medibot.

## Technical details

None.

## Media

image
image

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

None.

**Changelog**

:cl:
- tweak: Medibots now smile!

### CODE REVIEW

**Comment by deltanedas** [comment]
put it in deltav folder and add `sprite: DeltaV/Mobs/Silicon/Bots/medibot.rsi` to the medibot prototype with a comment

File: Resources/Textures/Mobs/Silicon/Bots/medibot.rsi/medibot.png
Comment (deltanedas) [inline Resources/Textures/Mobs/Silicon/Bots/medibot.rsi/medibot.png:1]: revert changes to the upstream texture now :trollface:

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Textures/Mobs/Silicon/Bots/medibot.rsi/medibot.png
Comment (Radezolid) [reply Resources/Textures/Mobs/Silicon/Bots/medibot.rsi/medibot.png:1]: I just removed it from the changes before this comment, i think it shouldn't affect it. At least the Files Changed tab doesn't mention the original and it looks fine on my PC.

Comment (deltanedas) [reply Resources/Textures/Mobs/Silicon/Bots/medibot.rsi/medibot.png:1]: youre right its fixed now, github was doing a funny and showed both of them

**Review by deltanedas — APPROVED** [review]

**Pull request merged**